charts/seed-bootstrap/charts/vpa/values.yaml

```diff
@@ -16,6 +16,9 @@ admissionController:
   podAnnotations: {}
   podLabels: {}
   enableServiceAccount: true
+  registerWebhook: false
+  caCert: abcd
+  controlNamespace: abcd
   registerByURL: false
```

Comment:
```suggestion
```

Can this one be removed now? We always want to create the webhook config ourselves, right?